addons/web/models/base_document_layout.py

```diff
@@ -304,3 +306,10 @@ def _compile_scss(self, scss_source):
             )
         except libsass.CompileError as e:
             raise libsass.CompileError(e.args[0])
+
+    @api.depends('company_details')
+    def _compute_empty_company_details(self):
+        # In recent change when an html field is empty a <p> balise remains with a <br> in it,
+        # but when company details is empty we want to put the info of the company
```

Comment:
Not a big issue, but this comment is a bit too temporal with the `In recent changes`. It will feel weird/wrong the older this gets

Also, it's not that recent, see this commit :wink: 
https://github.com/odoo/odoo/commit/974f512f5f5d3b9f80a8c3fcde290e4f55cf1230